app/controllers/users_controller.rb

```diff
@@ -25,14 +25,14 @@ def show
   end
 
   def new
-    @user = User.new
+    @user = User.new(user_params)
     @title = "Add a Staff Account"
     init_fields
   end
 
   def create
-    @user = User.new
-    @user.accessible = :all if current_user.admin?
+    @user = User.new(user_params)
+    # @user.accessible = :all if current_user.admin?
     @user.attributes = params[:user]
```

Comment:
this line can hit the bricks, your attributes are already set from `@user = User.new(user_params)`